parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -344,6 +418,7 @@ pub mod pallet {
 				ticket.message.as_bounded_slice(),
 				AggregateMessageOrigin::Parachain(ticket.origin),
 			);
+			Self::charge_fee(&ticket).map_err(|_| SubmitError::ChargeFeeFailed)?;
```

Comment:
As discussed on Slack, we should charge a static fee so that malicious parachains don't DOS the outbound-queue.

However we should use the mechanisms provided by the XCM executor to do this, which involves:

1. In our [implementation](https://github.com/Snowfork/snowbridge/blob/1ffd01ec450ba06c5d7562e431b03b50c3c55c2c/parachain/primitives/router/src/outbound/mod.rs#L140) of `ExportXcm::validate`, we must specify the fee for the export.
2. Wait until this PR is ready so that the XCM executor automatically handles the fee for us: https://github.com/paritytech/polkadot-sdk/pull/1234

We can still use the code in `charge_fee` for charging a deposit for using these permissionless APIS:
1. create_agent
2. create_channel

Reply:
https://github.com/Snowfork/snowbridge/pull/940/commits/c834c49190bad783ef1f98be04e703e137b9e6f9

@vgeddes Is your comment `charging a deposit` refers to some guarantee deposit something like the 10 DOT in `asset_hub` to create asset?